tests/test_cli/test_inputhandler/test_inputhandler.cpp

```diff
@@ -448,3 +442,35 @@ TEST(Address, getPubKey)
     EXPECT_EQ(adr.getPublicKey(),pubKeyFromPem);
 }
 
+//TODO: - Change this ugly tests in future ticket, because Transaction class is going to be completely changed.
+// -Move these tests in separate file in the tests ticket
+// -Add more tests for different signatures/transactions
+TEST(Signer, getSignature)
```

Comment:
Can be renamed to `getSignatureForArbitraryData`, and the other one: `getSignatureForTransaction`.

Reply:
This will definitely be renamed in the future PR with tests.